graylog2-web-interface/package.json

```diff
@@ -67,15 +67,17 @@
     "babel-plugin-react-transform": "^1.1.1",
     "clean-webpack-plugin": "^0.1.3",
     "css-loader": "^0.23.0",
-    "enzyme": "^1.4.1",
+    "enzyme": "^2.0.0",
     "eslint": "^2.2.0",
-    "eslint-config-airbnb": "6.0.2",
+    "eslint-config-airbnb": "^9.0.0",
```

Comment:
I can see why we want to keep airbnb's style guide more or less up-to-date, but honestly at the moment I feel against updating it.

First, every time this changes there are a bunch of rules which are annoying and we need to disable. For instance, after installing this new version all our internal imports are marked as red by eslint.

Additionally, I think it makes harder for us to achieve a decent styling across our project, because even the newly formatted code is again outdated. Before doing this update we still have over 500 styling issues (discarding the legacy folder) only in the web interface project.

My proposal would be to hold updates on the eslint config until we are in a good shape style-wise, at least in the web-interface project, and then we should enforce the linting check. Once we are there, I think it would be easy to update the configuration every now and then, and get a reasonable compromise between new rules, period of stability for the styling, and time wasted adapting code to it.